src/plugins/video/video.yml

```diff
@@ -7,6 +7,9 @@ source:
 externalLink:
   en: External link
   de: Externer Link
+internalLink:
```

Comment:
Actually, we do not need this. I realized the translation key should be "elmuCdn", not "internalLink" (that was a bug!)

Reply:
Fixed